src/serverdlg.cpp

```diff
@@ -193,7 +193,7 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
         SystemTrayIcon.setContextMenu ( pSystemTrayIconMenu );
 
         // set tool text
-        SystemTrayIcon.setToolTip ( QString ( APP_NAME ) + tr ( " server" ) );
+        SystemTrayIcon.setToolTip ( tr ( "%1 server", "%1 is the name of the main application" ).arg(APP_NAME) );
```

Comment:
```suggestion
        SystemTrayIcon.setToolTip ( tr ( "%1 server", "%1 is the name of the main application" ).arg( APP_NAME ) );
```
You should run clang-format on your pc this should fix the style.